test/unit/object/test_item.py

```diff
@@ -12,6 +12,30 @@ def test_item_and_response(test_file, test_folder, mock_file_response, mock_fold
     return test_folder, mock_folder_response
 
 
+@pytest.fixture(params=('empty', 'same', 'other'))
+def test_collections_for_addition(mock_collection_id, request):
```

Comment:
Just because it's not immediately obvious, can you add a really short docstring just giving a better idea of what the two tuple elements represent?

Same for the next fixture.